components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -358,7 +361,8 @@ && isStepHasMultiOption(context)
                 */
                 if (isBackToFirstStepRequest(request) || (isIdentifierFirstRequest(request)
                         && (!isFlowHandlerInCurrentStepCanHandleRequest(context, request)
-                        && !FrameworkUtils.isIdfInitiatedFromAuthenticator(context)))) {
+                        && !FrameworkUtils.isIdfInitiatedFromAuthenticator(context)))
```

Comment:
shall we revert this change

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/6647/commits/ffb0217e864a5bf0248aea542ae14d3e57475659